website/download/index.md

```diff
@@ -20,7 +20,7 @@ You can verify the download by following these [procedures](https://www.apache.o
 
 #### v2.5.0
 - This is a major release after 2.4, with 96 bug fixes and enhancement. Check the [v2.5.0 release announcement](/blog/2018/09/20/release-v2.5.0/) and the release notes. 
-- [Release notes](/docs/release_notes.html) and [upgrade guide](/docs/howto/howto_upgrade.html)
+- [Kylin Release Notes](/docs/releases/kylin_release_notes.html) and [Upgrade Kylin From Old Versions](/docs/howto/howto_upgrade.html)
```

Comment:
I think the "upgrade guide" in lower case is more friendly than "Upgrade Kylin From Old Versions" in hump...